packages/suite/src/components/suite/NotificationCard.tsx

```diff
@@ -22,34 +15,46 @@ interface NotificationCardProps {
     ['data-test']?: string;
 }
 
-const getMainColor = (variant: NotificationCardProps['variant'], theme: SuiteThemeColors) => {
+const getTextColor = (variant: NotificationCardProps['variant'], theme: DefaultTheme) => {
```

Comment:
all those functions should also be replaced by simply using the right Button variants.